src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,51 @@
+package nl.novi.techiteasy.controllers;
+
+import jakarta.validation.Valid;
+import nl.novi.techiteasy.dtos.RequestTelevisionDto;
+import nl.novi.techiteasy.models.Television;
+import nl.novi.techiteasy.services.TelevisionService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequestMapping("/televisions")
+@RestController
+public class TelevisionController {
+    private final TelevisionService televisionService;
+
+    public TelevisionController(TelevisionService televisionService) {
+        this.televisionService = televisionService;
+    }
+
+    //    GET-request voor alle televisies
+    @GetMapping()
+    public ResponseEntity<List<Television>> getTelevisions(){
```

Comment:
Je returned hier een `Lis<Television>`. We mogen van nu niet meer een entity object driect naar de client retourneren, we gebruiken daar nu een DTO voor. Je moet de Television dus nog naar een DTO zien te vertalen.